wpilibNewCommands/src/main/native/include/frc2/command/DeferredCommand.h

```diff
@@ -29,11 +29,12 @@ namespace frc2 {
 class DeferredCommand : public CommandHelper<Command, DeferredCommand> {
  public:
   /**
-   * Creates a new DeferredCommand that runs the supplied command when
-   * initialized, and ends when it ends. Useful for lazily
-   * creating commands at runtime. The supplier will be called each time this
-   * command is initialized. The supplier <i>must</i> create a new Command each
-   * call.
+   * Creates a new DeferredCommand that directly runs the supplied command when
+   * initialized, and ends when it ends. Useful for lazily creating commands
+   * when the DeferredCommand is reached in a composition, such as if the
```

Comment:
This note about "when the DeferredCommand is reached in a composition" should be changed to match Java's "when the DeferredCommand is initialized".